<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GCS93-PC" failures="1" tests="2" name="Error Test" time="52.471" errors="0" timestamp="2022-08-24T12:51:53 IST">
  <testcase classname="Appium.appiumframewokdesign.eCommerenceTest1" name="FillForm_ErrorMessage" time="8.313">
    <failure type="java.lang.AssertionError" message="expected [Please your name] but found [Please enter your name]">
      <![CDATA[java.lang.AssertionError: expected [Please your name] but found [Please enter your name]
at Appium.appiumframewokdesign.eCommerenceTest1.FillForm_ErrorMessage(eCommerenceTest1.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- FillForm_ErrorMessage -->
  <testcase classname="Appium.appiumframewokdesign.eCommerenceTest1" name="FillForm_PositiveFlow" time="17.659"/>
</testsuite> <!-- Error Test -->
